controllers/storagecluster/storageclient.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 const (
-	ocsClientConfigMapName = "ocs-client-operator-config"
-	manageNoobaaSubKey     = "manageNoobaaSubscription"
+	ocsClientConfigMapName         = "ocs-client-operator-config"
+	manageNoobaaSubKey             = "manageNoobaaSubscription"
+	csiControllerPluginHostNetwork = "csiCtrlPluginHostNetwork"
```

Comment:
@leelavg, any suggestions on the name?

Reply:
Naming looks fine. For format either `smallcase-with-hyphen` or `UPPERCASE_WITH_UNDERSCORE` would be good, can we go w/ one of that, can leave existing one (manageNoobaa) as I feel mixed case could be troublesome for manual editing 🤔.

Reply:
Based on pod names, `csi-ctrlplugin-host-network` would be good?

Reply:
changed to `csi-ctrlplugin-host-network`